xbmc/utils/StringUtils.cpp

```diff
@@ -512,6 +512,17 @@ std::string& StringUtils::TrimRight(std::string &str, const char* const chars)
   return str;
 }
 
+int StringUtils::ReturnDigits(std::string str)
+{
+  std::stringstream ss;
+  for (const auto& character : str)
+  {
+    if (isdigit(character))
+      ss << character;
+  }
+  return atoi(ss.str().c_str());
+}
+
```

Comment:
Votes field was string and that string could be formatted in several ways, for example
10000
10.000
10'000
10.000 votes
That's the safest way to get the votes. Here's the discussion:
https://github.com/xbmc/xbmc/pull/8062